packages/core/src/browser/tree/tree-widget.tsx

```diff
@@ -964,7 +964,7 @@ export class TreeWidget extends ReactWidget implements StatefulWidget {
      * @returns the CSS properties if available.
      */
     protected getDefaultNodeStyle(node: TreeNode, props: NodeProps): React.CSSProperties | undefined {
-        const paddingLeft = this.getPaddingLeft(node, props) + 'px';
+        const paddingLeft = (this.getPaddingLeft(node, props) + this.props.leftPadding) + 'px';
```

Comment:
Suggestion: Instead of appending the `props.leftPadding` value again, we should probably just change `getPaddingLeft` to `return (props.depth + 1) * ...`.

Reply:
Hi @msujew ,  
Thanks a lot for the suggestion, I will correct my code accordingly. 
Have a great day!